stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -232,6 +468,816 @@
+      "cybox": false
+    }
+  },
+  "container": {
+    "id": {
+      "key": "x_ecs_container.id",
+      "cybox": false
+    },
+    "image": {
+      "name": {
+        "key": "x_ecs_container.image_name",
+        "cybox": false
+      },
+      "tag": {
+        "key": "x_ecs_container.image_tag",
+        "cybox": false
+      }
+    },
+    "labels": {
+      "key": "x_ecs_container.labels",
+      "cybox": false
+    },
+    "name": {
+      "key": "x_ecs_container.name",
+      "cybox": false
+    },
+    "runtime": {
+      "key": "x_ecs_container.runtime",
+      "cybox": false
+    }
+  },
+  "dll":{
+    "name": {
+      "key": "x_ecs_dll.name",
+      "cybox": false
+    },
+    "path": {
+      "key": "x_ecs_dll.path",
+      "cybox": false
+    }
+  },
+  "dns": {
+    "answers": {
+      "class": {
+        "key": "x_ecs_dns.answers_class",
+        "cybox": false
+      },
+      "data": {
+        "key": "x_ecs_dns.answers_data",
+        "cybox": false
+      },
+      "name": {
+        "key": "x_ecs_dns.answers_name",
+        "cybox": false
+      },
+      "ttl": {
+        "key": "x_ecs_dns.answers_ttl",
+        "cybox": false
+      },
+      "type": {
+        "key": "x_ecs_dns.answers_type",
+        "cybox": false
+      }
+    },
+    "header_flags": {
+        "key": "x_ecs_dns.header_flags",
+        "cybox": false
+    },
+    "id": {
+      "key": "x_ecs_dns.id",
+      "cybox": false
+    },
+    "op_code": {
+      "key": "x_ecs_dns.op_code",
+      "cybox": false
+    },
+    "question": {
+      "class": {
+        "key": "x_ecs_dns.question_class",
+        "cybox": false
+      },
+      "name": {
+        "key": "x_ecs_dns.question_name",
+        "cybox": false
+      },
+      "registered_domain": {
+        "key": "x_ecs_dns.question_registered_domain",
+        "cybox": false
+      },
+      "subdomain": {
+        "key": "x_ecs_dns.question_subdomain",
+        "cybox": false
+      },
+      "top_level_domain": {
+        "key": "x_ecs_dns.question_top_level_domain",
+        "cybox": false
+      },
+      "type": {
+        "key": "x_ecs_dns.question_type",
+        "cybox": false
+      }
+    },
+    "resolved_ip": {
+      "key": "x_ecs_dns.resolved_ip",
+      "cybox": false
+    },
+    "response_code": {
+      "key": "x_ecs_dns.response_code",
+      "cybox": false
+    },
+    "type": {
+      "key": "x_ecs_dns.type",
+      "cybox": false
+    }
+  },
+  "ecs": {
+    "version": {
+      "key": "x_ecs.version",
+      "cybox": false
+    }
+  },
+  "error": {
+    "code": {
+      "key": "x_ecs_error.code",
+      "cybox": false
+    },
+    "id": {
+      "key": "x_ecs_error.id",
+      "cybox": false
+    },
+    "message": {
+      "key": "x_ecs_error.message",
+      "cybox": false
+    },
+    "stack_trace": {
+      "key": "x_ecs_error.stack_trace",
+      "cybox": false
+    },
+    "type": {
+      "key": "x_ecs_error.type",
+      "cybox": false
+    }
+  },
+  "file": {
```

Comment:
There a lot of ECS file fields that aren't being captured here. It would be good to add them to the STIX file objects as custom properties, like `file.x_ecs_uid`

Reply:
I think I included all. let me know if I missed any